assets/javascripts/discourse/services/chat.js

```diff
@@ -52,6 +52,7 @@ export default class Chat extends Service {
   directMessagesLimit = 20;
   _chatOpen = false;
   _fetchingChannels = null;
+  isNetworkUnreliable = false;
```

Comment:
I might make it into its own service at some point and toggle it's value from more ajax calls

Reply:
Yeah, this seems like a service we would want to have in core, eventually.